compiler/testData/diagnostics/testsWithStdLib/contracts/controlflow/flowInlining/breakContinuesInInlinedLambda.fir.kt

```diff
@@ -33,12 +33,12 @@ fun test() {
                 // Loop executed exectly once, initializing x
                 myRun { x.inc() }
 
-                myRun { x = 42 }
+                myRun { <!VAL_REASSIGNMENT!>x<!> = 42 }
                 break
             }
             // x is I?D here because loop could've been execited
             // VAL_REASSIGNMENT isn't reported because of repeating diagnostic
-            x = 42
+            <!VAL_REASSIGNMENT!>x<!> = 42
```

Comment:
Aha, very clear evidence here. According to the comment, the repeating diagnostic error is not reported. So, this one is `VAL_REASSIGNMENT`, but old FE somehow collects and reports an error per error kind maybe.